docs/api/app.md

```diff
@@ -601,7 +632,7 @@ Hides all application windows without minimizing them.
 
 ### `app.isHidden()` _macOS_
 
-Returns `boolean` - `true` if the application—including all of its windows—is hidden (e.g. with `Command-H`), `false` otherwise.
+Returns `boolean` - `true` if the application-including all of its windows???is hidden (e.g. with `Command-H`), `false` otherwise.
```

Comment:
```suggestion
Returns `boolean` - `true` if the application - including all of its windows - is hidden (e.g. with `Command-H`), `false` otherwise.
```

Reply:
Hello @felixrieseberg thank you for the remark. 

Honestly due to continuous work of community into notifications module my branch becomes conflicting, and I have to resolve conflicts, sometimes in parts which are not related to notifications at all, this is a result of one of such resolving. Maybe I missed this moment, will take a look when have enough time.